cleverhans/utils.py

```diff
@@ -99,8 +99,6 @@ def to_categorical(y, num_classes=None):
     :return: A binary matrix representation of the input.
     """
     y = np.array(y, dtype='int').ravel()
-    if not num_classes:
```

Comment:
Can you leave this if statement for now, and add a warning that it will be removed?